Cargo.toml

```diff
@@ -27,6 +27,7 @@ strict_encoding_support = ["strict_encoding"]
 hex = "0.4.3"
 hex-literal = "0.3.1"
 keccak-hash = "0.7.0"
```

Comment:
I'm not against adding `tiny-keccak` but you could just use `keccak-hash` like we did for bulletproof. `keccak-hash` uses `tiny-keccak` under the hood anyway, so the dependency footprint would actually be the same. I don't know.

Reply:
Yeah, I am thinking of throwing out `keccak-hash` but because the API is actually inconvenient for our usecase and a utility function that hashes a single slice can easily be added locally.

See https://github.com/comit-network/monero-rs/issues/12.